### ISSUE TITLE
Make so you can change size and font color of the ' top ' bar <3 and, custom icons . .

### ISSUE THREAD

**Issue by jasperbrooks79** [godotengine/godot-proposals#2240]
### Describe the project you are working on
A Tomb raider game <3 <3

Character ::O
![Skærmbillede (983)](https://user-images.githubusercontent.com/61069740/106865899-16fd6980-66cc-11eb-9ac9-26063e8ede56.png)

### Describe the problem or limitation you are having in your project
I'm super - happy for the editor, interface is GLORIOUS <3 But, I was wondering if there could be customization option, for size and, font of these letters . .
<img width="682" alt="2021-02-04 0935" src="https://user-images.githubusercontent.com/61069740/106866076-51ff9d00-66cc-11eb-9cbc-c4f3d8be2f31.png">

Also, scale of the small icons, so I can maybe make this <3 . .
![2021-02-04 0936](https://user-images.githubusercontent.com/61069740/106866259-87a48600-66cc-11eb-93c8-785b99d7d5a3.jpg)

I want to make an interface, that resembles a classic RPG / or action game look, where one can select ' Stats ', ' Inventory ', ' Quests ', so on, at the top <3

![2021-02-04 0939](https://user-images.githubusercontent.com/61069740/106866496-d05c3f00-66cc-11eb-8e1d-173b274fdf43.jpg)

![2021-02-04 0938](https://user-images.githubusercontent.com/61069740/106866504-d4885c80-66cc-11eb-9d51-9096eb41c767.jpg)

When I open Godot first time, it was a bit confusing, I know now those letters, 2D, 3D, Script, are VERY important, maybe one could have some settings, for icons, small icon size, medium icon size, etc . . That way, I could make my dream editor, and I think they should be a bit bigger, I thought they were ' not ' important, over-looked them for a long time, not sure :OO for beginners, it'd help see, I should click those, and find what they do, since they're big, and I can realize dream, of making my ' tuned ' version, of editor, look like classic RPG / action game, so using Godot, reminds me of all my favorite games and, genres <3 <3

### Describe the feature / enhancement and how it helps to overcome the problem or limitation
Add a customization feature, to set the ' size ' of 2D, 3D, to a custom font, with custom size and, to have small, medium, large, and x - large and, xx - large, also for people with sight problems, or accessibility or, so . . .

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams
Custom font, for those letters, and size . . <3
<img width="1040" alt="2021-02-04 0943" src="https://user-images.githubusercontent.com/61069740/106867028-6d1edc80-66cd-11eb-98fb-5182024d9235.png">

also, add support for a ' shadow ' effect, like in using custom fonts, in UI work . . So, one can make an ' outline ' or, so <3 it'd really make the editor pop, and that's my final request, for a ' better ' UI, more user - friendly and, cool <3

### If this enhancement will not be used often, can it be worked around with a few lines of script?
I don't know how write code, to make different engine :(( . .

### Is there a reason why this should be core and not an add-on in the asset library?
Well, it'd just be a customization option, and people might make their dream Godot . . Like . . .

also, maybe to use custom icons, or load one's own set, or so . . :OO 

here, easier to see, the small letters >> 
<img width="1920" alt="Skærmbillede (984)" src="https://user-images.githubusercontent.com/61069740/106867622-1e257700-66ce-11eb-95fb-d5f4018ac52d.png">

also, for new users, it'd be ' obvious ' or, so, to click those, and see what they do, to get started, us noobs <3 . . .

then, in AssetLib, one could download some new icons, from users . . :OO :OO . .

also, maybe vector icons, that can scale, support for that . . <3

**Comment by pycbouh** [comment]
I don't think it makes much sense as an option, but we could add a definition to the editor theme so it can be overridden with a custom editor theme, which users can already do. The icons can also be overridden as a part of the theme.

**Comment by jasperbrooks79** [comment]
yes, if I change the editor font, the entire editor is changed, I wish to only change those 2D, 3D, to a bigger size, I use 4K, and actually have trouble, sometimes, seeing them, or so :OO . .

I didn't know one can make icons different :O is it, in settings Thx . . <3

**Comment by pycbouh** [comment]
> I checked editor settings, you can't change icon, or ?? . . :O

It is, but it's not a dedicated setting. You can customize the entire editor theme using Godot's Theme resource:
![image](https://user-images.githubusercontent.com/11782833/106904418-37e5af00-670c-11eb-8c99-4c1e2b65d750.png)

Every property defined for a theme can be overridden, and the editor theme has a bunch of custom properties defining its looks. To get started you can create a new Theme resource, open the Theme editor from the bottom panel and pick "Create Empty Editor Template":

![image](https://user-images.githubusercontent.com/11782833/106904792-a9256200-670c-11eb-89d2-36c04af3a283.png)

**Comment by jasperbrooks79** [comment]
Ok, it's a very complex editor, it gives control over every, every detail . .

I realize it's a lot, but I do wish we had option of, editing visual script, some common stuff, same way code 'editor' is, very changing like, one can set color, of almost everything, without having to create a new theme also, I have to restart editor every time I change theme, and it takes a, quite some time <3

**Comment by pycbouh** [comment]
> I realize it's a lot, but I do wish we had option of, editing visual script, some common stuff, same way code 'editor' is, very changing like, one can set color, of almost everything, without having to create a new theme

Well, syntax highlighting is a very individual thing among coders and is indeed easily configurable because of that. But the style of the top bar text is hardly "common stuff". This is exactly the type of detail you'd edit in a more compelete and complex way rather than expose it with a simple setting, because realistically nobody would need to change it.

**Comment by jasperbrooks79** [comment]
well, okay, it works well, as it is . . Just wish could make my own custom Godot, felt like playing a video game, when MAKING a game <3 . . but, it's currently possible in ' Theme ' node <3 <3 . . .

**Comment by SIsilicon** [comment]
To clarify, `Theme` inherits from `Resource`; not `Node`.

**Comment by pycbouh** [comment]
> Edit: Also, didn't you already make a proposal for changing the top icons here? -> https://github.com/godotengine/godot-proposals/issues/2108

I thought I remembered this already being proposed! Thanks, I'll close the other one, as this one at least has some answers
